### PR TITLE
Feature dynamic tokens

### PR DESCRIPTION
**Pull request by bansri29**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,7 +43,10 @@ vault_server:
 
 vault_plugin:
 	vault secrets enable -path=astra vault-plugin-secrets-datastax-astra
-	vault write astra/config org_id="$ORG_UUID" astra_token="$TOKEN" url="https://api.astra.datastax.com" logical_name="org_logical_name"
+	# vault write astra/config org_id="$ORG_UUID" astra_token="$TOKEN" url="https://api.astra.datastax.com" logical_name="org_logical_name"
+	vault write astra/config org_id="03acd0a6-1451-4827-b206-81ad1099f1a1" astra_token="AstraCS:oWGwZjPlKzmYNTSjZXrTLmLp:3991c593265d8a7437571172a3dae19451256acd8d682c6616f545c2b67c1f5a" url="https://api.astra.datastax.com" logical_name="org1"
+	sh update_roles.sh
+	vault write astra/org/token org_id="03acd0a6-1451-4827-b206-81ad1099f1a1" role_name="api_r_w_user" logical_name="org1"
```
Comment (vaishakhbn) [inline Makefile:49]: can you remove these tokens from your org and not expose any actual org/token details here?

**Comment by vaishakhbn** [comment]
Don't push binary files either.

File: Makefile (hunk 1 above)
Comment (bansri29) [reply Makefile:49]: done

**Review by vaishakhbn — APPROVED** [review]

**Comment by vaishakhbn** [comment]
add an appropriate commit message & description about what this PR is doing. This isn't "Feature dynamic tokens". Instead, it is support for reading tokens by providing clientID

**Comment by bansri29** [comment]
The changes made are to provide support for reading tokens by providing clientID. Users can execute “vault read astra/org/token ” without additional parameters to retrieve a token.

**Pull request merged**